Doppler.HtmlEditorApi/Infrastructure/Repository.cs

```diff
@@ -24,7 +24,7 @@ public async Task<ContentModel> GetCampaignModel(string accountName, int campaig
 JOIN [User] u ON u.IdUser = ca.IdUser
 WHERE co.IdCampaign = @campaignId  AND u.Email = @accountName AND co.EditorType = 5";
                 var result = await connection.QueryFirstOrDefaultAsync<ContentRow>(databaseQuery, new { campaignId, accountName });
```

Comment:
Ooops, I am not sure if it will work: https://github.com/DapperLib/Dapper/issues/1571

@maurocardinali let me know when you be testing your changes and we could test it too.